DQMOffline/Trigger/python/HiggsMonitoring_Client_cff.py

```diff
@@ -1,4 +1,189 @@
+        "effic_eleEta_1       'efficiency vs electron eta; electron eta ; efficiency' eleEta_1_numerator       eleEta_1_denominator",
+        "effic_elePhi_1       'efficiency vs electron phi; electron phi ; efficiency' elePhi_1_numerator       elePhi_1_denominator",
+        "effic_elePt_1_variableBinning       'efficiency vs electron pt; electron pt [GeV]; efficiency' elePt_1_variableBinning_numerator       elePt_1_variableBinning_denominator",
+        "effic_eleEta_1_variableBinning       'efficiency vs electron eta; electron eta ; efficiency' eleEta_1_variableBinning_numerator       eleEta_1_variableBinning_denominator",
+        "effic_elePtEta_1       'efficiency vs electron pt-#eta; electron pt [GeV]; electron #eta' elePtEta_1_numerator       elePtEta_1_denominator",
+        "effic_eleEtaPhi_1       'efficiency vs electron #eta-#phi; electron #eta ; electron #phi' eleEtaPhi_1_numerator       eleEtaPhi_1_denominator",
+    ),
+)
+
+######Double Electron + Single Muon######
+
+mu8diEle12CaloIdLTrackIdL_effele = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/TriLepton/HLT_Mu8_DiEle12_CaloIdL_TrackIdL_muegRef/"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_elePt_2       'efficiency vs electron pt; electron pt [GeV]; efficiency' elePt_2_numerator       elePt_2_denominator",
+        "effic_eleEta_2       'efficiency vs electron eta; electron eta ; efficiency' eleEta_2_numerator       eleEta_2_denominator",
+        "effic_elePhi_2       'efficiency vs electron phi; electron phi ; efficiency' elePhi_2_numerator       elePhi_2_denominator",
+        "effic_elePt_2_variableBinning       'efficiency vs electron pt; electron pt [GeV]; efficiency' elePt_2_variableBinning_numerator       elePt_2_variableBinning_denominator",
+        "effic_eleEta_2_variableBinning       'efficiency vs electron eta; electron eta ; efficiency' eleEta_2_variableBinning_numerator       eleEta_2_variableBinning_denominator",
+        "effic_elePtEta_2       'efficiency vs electron pt-#eta; electron pt [GeV]; electron #eta' elePtEta_2_numerator       elePtEta_2_denominator",
+        "effic_eleEtaPhi_2       'efficiency vs electron #eta-#phi; electron #eta ; electron #phi' eleEtaPhi_2_numerator       eleEtaPhi_2_denominator",
+    ),
+) 
+
+mu8diEle12CaloIdLTrackIdL_effmu = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/TriLepton/HLT_Mu8_DiEle12_CaloIdL_TrackIdL_dieleRef/"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_muPt_1       'efficiency vs muon pt; muon pt [GeV]; efficiency' muPt_1_numerator       muPt_1_denominator",
```

Comment:
This is the reference from DiElectron trigger. So we monitor the other leg(muon).